src/gui/main_gui.cpp

```diff
@@ -204,12 +197,15 @@ GUITest(World world) {
     MR.set_light_direction(light_direction);
     MR.set_depth_projection_matrix(depth_projection_matrix);
 
-    for (auto& m : chunk_meshes) {
-        MR.add_mesh(std::make_shared<terrain::TerrainMesh>(m));
+    for (auto m : chunks_mesh) {
+        MR.add_mesh(m);
     }
     MR.add_mesh(std::make_shared<terrain::StaticMesh>(treesMesh));
     MR.set_depth_texture(SM.get_depth_texture());
 
+    unsigned int frame_id = 0;
+    bool do_set = false;
```

Comment:
This needs a better name

And its also never set to true

Reply:
used to test, this is manualy set to true.